main.go

```diff
@@ -260,11 +262,12 @@ type Issue struct {
 }
 
 func (i *Issue) String() string {
-	col := ""
-	if i.Col != 0 {
-		col = fmt.Sprintf("%d", i.Col)
-	}
-	return fmt.Sprintf("%s:%d:%s:%s: %s (%s)", strings.TrimSpace(i.Path), i.Line, col, i.Severity, strings.TrimSpace(i.Message), i.Linter)
+	tmpl, err := template.New("output").Parse(*formatFlag)
```

Comment:
It would be better to make this a global, possibly via an `Action()` on the formatFlag.